pkg/loadbalancer/server.go

```diff
@@ -208,9 +208,10 @@ func (s *Server) createLoadBalancer(clusterName string, service *v1.Service, ima
 		// including some ones docker would otherwise do by default.
 		// for now this is what we want. in the future we may revisit this.
 		"--privileged",
-		"--restart=on-failure",                   // to deal with the crash casued by https://github.com/envoyproxy/envoy/issues/34195
-		"--sysctl=net.ipv4.ip_forward=1",         // allow ip forwarding
-		"--sysctl=net.ipv4.conf.all.rp_filter=0", // disable rp filter
+		"--restart=on-failure",                           // to deal with the crash casued by https://github.com/envoyproxy/envoy/issues/34195
+		"--sysctl=net.ipv4.ip_forward=1",                 // allow ip forwarding
+		"--sysctl=net.ipv4.conf.all.rp_filter=0",         // disable rp filter
+		"--sysctl=net.ipv4.ip_unprivileged_port_start=1", // Allow lower port numbers for podman (see https://github.com/containers/podman/blob/main/rootless.md for more info)
```

Comment:
@BenTheElder I imagine this option should be harmless for the rest of the runtime or should this gate it behind the `KIND_EXPERIMENTAL_PROVIDER=podman` env variable

Reply:
I think you'd know better than me, IMHO we should avoid differential behavior as much as possible, and I can't htink yet why we'd have a problem permitting this. The container is already privileged.